src/app/globals.css

```
@@ -231,4 +231,103 @@
       transform: translateY(-50%);
     }
   }
-}
\ No newline at end of file
+}
+/* Markdown rendering for assistant chat messages */
+.markdown {
+  font-size: 0.875rem;
+  line-height: 1.6;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)